openwisp_radius/base/forms.py

```diff
@@ -43,7 +43,6 @@ class RadiusCheckForm(ModeSwitcherForm):
         label=_('Value'),
         required=False,
         max_length=radcheck_value_field.max_length,
-        widget=forms.PasswordInput(),
```

Comment:
Why this field is converted to CharField

Reply:
My bad. I thought it was no longer needed.